src/sass/_Fabric.Icons.scss

```diff
@@ -400,4 +400,21 @@
     transform-origin: 50% 50%;
     z-index: $ms-zIndex-back;
   }
-}
\ No newline at end of file
+}
+
+// Fabric Core Icon variables
+$ms-Icon-12: 12px;
+$ms-Icon-16: 16px;
+$ms-Icon-20: 20px;
+
+.ms-Icon-12 {
+  font-size: 12px;
```

Comment:
I think you meant to use the variables from above here, right? Also, these should be modifier classes like `ms-Icon--12` to be used in conjunction with `ms-Icon`.